main.js

```diff
@@ -0,0 +1,105 @@
+/* eslint no-restricted-syntax: "off", curly: "off" */
+/* eslint no-use-before-define: "off", curly: "off" */
+/* eslint no-unused-vars: "off", curly: "off" */
+/* eslint func-names: "off", curly: "off" */
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+
+  this.info = function () {
+    return (`This book is called ${this.title} from ${this.author} and it has ${this.pages} pages and${this.convert()}`);
+  };
+
+  this.convert = function () {
+    if (this.read) {
+      return ' this book has been read';
+    }
+
+    return ' this book has NOT been read';
+  };
+}
+
+const myLibrary = [];
+
+function printLibrary() {
+  document.getElementById('printing').innerHTML = '';
+
+  for (const i in myLibrary) {
+    if (i) {
+      const j = document.createElement('li');
+
+      const u = document.createElement('p');
+      u.innerHTML = myLibrary[i].info();
+
+      // delete button
+      const button = document.createElement('button');
+      button.classList.add('btn', 'btn-danger');
+      button.onclick = function () { deleteBook(i); };
+      button.innerHTML = 'Delete';
+
+      // change status button
+      const button2 = document.createElement('button');
+      button2.classList.add('btn', 'btn-primary', 'ml-2');
+      button2.onclick = function () { changeStatus(i); };
+      button2.innerHTML = 'Change status';
+
+      j.classList.add('list-group-item');
+      j.id = `book${i}`;
+      j.appendChild(u);
+      j.appendChild(button);
+      j.appendChild(button2);
+      document.getElementById('printing').appendChild(j);
+    }
+  }
+}
+
+function addBookToLibrary(book) {
+  myLibrary.push(book);
+  printLibrary();
+}
+
+function deleteBook(bookId) {
+  (myLibrary.splice(bookId, 1));
+
+  printLibrary();
+}
+
+function changeStatus(bookId) {
+  if (myLibrary[bookId].read) {
+    myLibrary[bookId].read = false;
+  } else {
+    myLibrary[bookId].read = true;
+  }
+
+  printLibrary();
+}
+
+function NewBook() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const pages = document.getElementById('pages').value;
+  const read = document.getElementById('read').checked;
```

Comment:
- [ ] **(OPTIONAL)** The form used to create a book should have validations, you should not be able to create an empty book.